main.js

```diff
@@ -563,9 +563,12 @@ function dirtyWords (wordToDefine) {
 	  }
 	else {
     	console.log('Urban Dictionary Definition:', body);
-    	var sample = JSON.parse(body);	
+    	var sample = JSON.parse(body);
+		if(sample != "{"list":[]}"{
```

Comment:
this will always evaluate falsey because `sample` is a JSON object and `"{"list":[]}"` is a string, so they're never be equivalent. This line also won't compile because there's a syntax error